### PR TITLE
Fix latest phpstan issues

### PR DESCRIPTION
**Pull request by soyuka**
Fix latest phpstan issues

### CODE REVIEW

File: tests/Behat/DoctrineContext.php [hunk 1]
```diff
@@ -198,17 +197,19 @@ public function __construct(ManagerRegistry $doctrine, UserPasswordEncoderInterf
...
         $this->schemaManager = $this->manager instanceof DocumentManager ? $this->manager->getSchemaManager() : null;
-        $this->classes = $this->manager->getMetadataFactory()->getAllMetadata();
     }
 
     /**
      * @BeforeScenario @createSchema
      */
     public function createDatabase()
     {
+        /** @var array<int, \Doctrine\ORM\Mapping\ClassMetadata> $classes */
```
Comment (alanpoulain) [inline tests/Behat/DoctrineContext.php:207]:
```suggestion
        /** @var list<\Doctrine\ORM\Mapping\ClassMetadata> $classes */
```

Tested: works!

File: tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php [hunk 2]
```diff
@@ -165,6 +165,7 @@ public function testFindIriBy(): void
          * @var EntityManagerInterface
          */
         $manager = self::$container->get('doctrine')->getManager();
+        /** @var list<int, \Doctrine\ORM\Mapping\ClassMetadata> $classes */
```
Comment (alanpoulain) [inline tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php:168]:
```suggestion
        /** @var list<\Doctrine\ORM\Mapping\ClassMetadata> $classes */
```

File: src/Bridge/Doctrine/Orm/Util/QueryBuilderHelper.php [hunk 3]
```diff
@@ -186,6 +186,7 @@ public static function getExistingJoin(QueryBuilder $queryBuilder, string $alias
      */
     private static function mapRootAliases(array $rootAliases, array $rootEntities): array
     {
+        /** @var bool|array $aliasMap */
```
Comment (guilliamxavier) [inline src/Bridge/Doctrine/Orm/Util/QueryBuilderHelper.php:189]: why not `array|false`?

File: tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php (hunk 2 above)
Comment (guilliamxavier) [reply tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php:168]: what's wrong with `\Doctrine\ORM\Mapping\ClassMetadata[]`?

**Pull request merged**